### ISSUE TITLE
Custom Loss function

### ISSUE THREAD

**Issue by the-nilosch** [dfuhge/Animate_SVG_v2#5]
As our embedding is uniquely interpreted, a manual loss function is needed.
The embedding is divided in the following parts

- 256 Features with DeepSVG Embedding --> Loss for simple float embeddings
- 7 Features, that are one-hot encoded --> Loss for One-Hot
  - Type "EOS" (1)
  - Animation Type (6)
- Parameters of animation (6) --> Only relevant parameters that are relevant to the true animation should be rated! So if ground truth is `-1` this should be no loss

Prerequisite
- Softmax is implemented in Transformer
